safety/safety.py

```diff
@@ -87,17 +88,31 @@ def fetch_database_url(mirror, db_name, key, cached, proxy):
         if cached_data:
             return cached_data
     url = mirror + db_name
-    r = requests.get(url=url, timeout=REQUEST_TIMEOUT, headers=headers, proxies=proxy)
+
+    try:
+        r = requests.get(url=url, timeout=REQUEST_TIMEOUT, headers=headers, proxies=proxy)
+    except requests.exceptions.ConnectionError:
+        raise NetworkConnectionError()
+    except requests.exceptions.Timeout:
+        raise RequestTimeoutError()
+    except requests.exceptions.RequestException:
+        raise DatabaseFetchError()
```

Comment:
Good thing you decided to add our own custom error classes here instead of just letting **requests** exceptions to slip through. Can you please pass the original exception as an `inner` attribute here?